patches/0007-Add-informative-backend-configuration-errors.patch

```diff
@@ -0,0 +1,302 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
True, but I'm actually thinking it might be better to go with the `go test [...] -update` approach and keep it inside the Go submodule for simplicity (depending on how you look at it).